src/mixins/filing-template-mixin.ts

```diff
@@ -359,6 +369,10 @@ export default class FilingTemplateMixin extends Mixins(DateMixin) {
       filing.changeOfRegistration.business.naics = this.getCurrentNaics
     }
 
+    if (this.hasOrgPersonChanged) {
```

Comment:
I'd move the `parties` code above inside this statement, so it only gets executed if needed instead of always.

Reply:
Updated